### ISSUE TITLE
Add localisation support like Bunny

### ISSUE THREAD

**Issue by Felitendo** [revenge-mod/revenge-rewrite#3]
Bunny supports other languages through crowdin with i18n:

https://github.com/bunny-mod/i18n

It would be nice if you could add that to revenge-rewrite and revenge-managed too :)

**Comment by PalmDevs** [comment]
For Revenge Manager, please open a separate issue there. I'll keep this in the backlog for now and work on it once other stuff is finished.
